cli/cmd/version.go

```diff
@@ -10,7 +10,8 @@ import (
 )
 
 // This variable is used by the "version" command and is set during build.
-var Version = "undefined"
+// Defaults to a safe value if not set.
+var Version = "v0.0.1"
```

Comment:
may be use 2 here ? since it has multi arch support ?

Reply:
ok

Reply:
GH is not showing this as updated to v0.0.2, but it is: https://github.com/rbtr/retina/commit/836040d6a9045ac6f03f0d2e664a6f14ca9dddb7#diff-e6043c8e0507d58cc731bbc33524ebec7431445e86bee4a7da52abb081669b38R14